src/main/java/com/google/cloud/tools/maven/deploy/AppDeployer.java

```diff
@@ -176,6 +176,7 @@ public void deployQueue() throws MojoExecutionException {
     DeployConfiguration buildDeployConfiguration(List<Path> deployables) {
       return DeployConfiguration.builder(deployables)
           .bucket(deployMojo.getBucket())
+          .gcloudMode(deployMojo.getGcloudMode())
```

Comment:
If the configuration is missing, will `getGcloudMode()` return null or an empty string? In either case, will it be safe?

Reply:
yeah it's safe to pass null, all config params can be null.